Moderation-Policy.md

```diff
@@ -197,6 +197,8 @@ of the [Code of Conduct][].
   indefinitely, and reporting accounts to GitHub.
 * Accounts that are reasonably believed to be bots (other than bots authorized
   by both TSC and CommComm) are subject to immediate Blocking.
+* Collaborators may use the Hide feature in the GitHub interface for off-topic
```

Comment:
to clarify, doing so would still require a post in moderation to notify others of doing it? (there's no paper trail for hiding; while there is now for edits and deletions)

Reply:
I'd prefer not. Hiding is easily reversible in ways editing and deleting are not (or at least didn't used to be if they are now). So I'd prefer to only add that overhead if it proves necessary.

Reply:
Oh i meant after the fact; not asking about it beforehand. Otherwise there’s no way to know who did the hiding.

Reply:
I'd still like to avoid that overhead, but if you or anyone else think it's necessary, I can live with that. I'd prefer to keep the requirements minimal and only scale them up if it proves necessary.

Reply:
Makes sense in general - most of the time it's pretty obvious and noncontroversial, and in those cases it's nice to avoid the overhead - but when a hiding reason is contentious it might warrant some discussion.

Reply:
Yes, any action that might be contentious should require notification in `moderation` repo or other conspicuous indication of the action. I would support a sentence saying so in the Moderation Policy that applies to all actions (but I would also consider that out of scope for this PR).